packages/salesforcedx-vscode-lwc/esbuild.config.js

```diff
@@ -19,13 +19,20 @@ const sharedConfig = {
     'applicationinsights',
     '@salesforce/lightning-lsp-common',
     '@salesforce/lwc-language-server',
-    '@babel/preset-typescript/package.json'
+    '@babel/preset-typescript/package.json',
+    'jest-editor-support'
```

Comment:
some artifacts in this dependency are not able to be bundled. It was never externalized because the extension still worked fine even if esbuild reports warnings every time. But I think we should move this out from now on.
<img width="972" alt="image" src="https://github.com/user-attachments/assets/7c2668f7-07d8-4600-b84d-8eaa87762250">

Reply:
the extensions size of lwc increases from 10.2MB to 12.8MB by externalizing `jest-editor-support`